src/Microsoft.AspNetCore.Mvc.ViewFeatures/ViewFeatures/IModelExpressionProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq.Expressions;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+
+namespace Microsoft.AspNetCore.Mvc.Rendering
```

Comment:
This and `ModelExpressionProvider` should be in the less-commonly used `Microsoft.AspNetCore.Mvc.ViewFeatures` namespace. Move `ModelExpression` as well. Then all three will have the same namespace and that will match their directory. (Could optionally place `ModelExpressionProvider` in `Microsoft.AspNetCore.Mvc.ViewFeatures.Internal` but I don't think it's worth hiding.)